pyuvsim/source.py

```diff
@@ -241,3 +243,39 @@ def get_size(self):
         Estimate the SkyModel memory footprint in bytes
         """
         return self.Ncomponents * self._basesize
+
+
+def read_hdf5(hdf5_filename, freq_ind):
```

Comment:
if this is specific to healpix, I'd suggest using that in the function name. Many different things can be represented in hdf5 files, including visibilities!

Reply:
This is to read a specific hdf5 format for sky models in `healvis`. It's included mainly because we don't have a well-defined file format yet for HEALPix shells.

Reply:
That's fine, I'm just objecting to the function name -- it should make it clear that it's for healvis's hdf5 rather than any generic hdf5.